app/views/submissions/_grades.html.erb

```diff
@@ -6,10 +6,10 @@
       <div class="problem_name"> <%= p.name.capitalize %>: </div>
       <div class="problem_score">
         <% if @cud.instructor? or @cud.course_assistant? %>
-          <b class="student_score" data-problem-id="<%= p.id %>" data-submission-id="<%= @submission.id %>"> <%= p_score && p_score.score ? p_score.score : raw("&ndash;") %> </b>
+          <b class="student_score" data-problem-id="<%= p.id %>" data-submission-id="<%= @submission.id %>"> <%= p_score&.score ? sprintf("%.1f", p_score.score) : raw("&ndash;") %> </b>
```

Comment:
Should we also change this to .2f and score.round(2) for consistency?

Reply:
Good catch, will do